.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,24 @@
+name: continuous-integration
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Setup python
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dev dependencies
+      run: |
+        pip install --upgrade pip wheel
+        pip install -e .[dev,test]
+    - name: Test with pytest
```

Comment:
Currently, or at least previously, codecov would fail CI if coverage decreased. I'm happy to stick with this for now, though a minimum coverage would be good in future.